build.sbt

```diff
@@ -659,11 +660,15 @@ lazy val `quill-cassandra-lagom` =
         Seq(
           "org.scala-lang.modules" %% "scala-collection-compat" % "2.4.4",
           "com.lightbend.lagom" %% "lagom-scaladsl-persistence-cassandra" % lagomVersion % Provided,
-          "com.lightbend.lagom" %% "lagom-scaladsl-testkit" % lagomVersion % Test
+          "com.lightbend.lagom" %% "lagom-scaladsl-testkit" % lagomVersion % Test,
+          "com.datastax.cassandra" %  "cassandra-driver-core" % "3.7.2",
+          // lagom uses datastax 3.x driver - not compatible with 4.x in API level
+          "io.getquill" %% "quill-cassandra" % "3.10.0" % "compile->compile"
         ) ++ versionSpecificDependencies
       }
     )
-    .dependsOn(`quill-cassandra` % "compile->compile;test->test")
+    //.dependsOn(`quill-cassandra` % "compile->compile;test->test")
+     .dependsOn(`quill-core`.jvm % "compile->compile;test->test")
```

Comment:
Take this out too. Only the dependency on `quill-cassandra` needs to be there.

Reply:
it would cause the problem:
`[error] /app/quill-cassandra-lagom/src/test/scala/io/getquill/context/cassandra/EncodingSpecHelper.scala:6:8: object Spec is not a member of package io.getquill
[error] import io.getquill.Spec
[error]        ^
[error] /app/quill-cassandra-lagom/src/test/scala/io/getquill/context/cassandra/EncodingSpecHelper.scala:8:43: not found: type Spec
[error] abstract class EncodingSpecHelper extends Spec {
[error]                                           ^
[error] /app/quill-cassandra-lagom/src/test/scala/io/getquill/context/cassandra/EncodingSpecHelper.scala:17:10: value mustEqual is not a member of Int
[error]     e.id mustEqual a.id
[error]          ^
[error] /app/quill-cassandra-lagom/src/test/scala/io/getquill/context/cassandra/EncodingSpecHelper.scala:19:10: value mustEqual is not a member of String
[error]     e.v1 mustEqual a.v1
`